selinon/trace.py

```diff
@@ -13,6 +13,10 @@
 +----------------------------+-------------------------------------+-----------------+------------------------------------+
 | Event                      |  Event description                  | Emitter         |  msg_dict.keys()                   |
 +============================+=====================================+=================+====================================+
+|                            | A flow has been scheduled.          |                 |  dispatcher_id, elective,          |
```

Comment:
typo elective->selective

Reply:
Awesome! Thanks for the review. If everything's fine now, I will fix unrelated pylint issues and proceed with this one.